app/views/core/CreateAccountModal/BasicInfoView.coffee

```diff
@@ -283,6 +283,8 @@ module.exports = class BasicInfoView extends CocoView
       else
         console.error 'BasicInfoView form submission Promise error:', e
         @state.set('error', e.responseJSON?.message or 'Unknown Error')
+        if @id == 'single-sign-on-confirm-view' and @signupState.get('path') is 'teacher'
+          window.tracker?.trackEvent 'Error in ssoConfirmView', {category: 'Teachers', label: @state.get('error')}
```

Comment:
Couple questions here:

1. Do we have anything like Sentry for collecting client side JS errors?  Debugging this may go much faster it will enrich the error reports with additional data and aggregate the errors we get.
1. Looks like `state.error` is set above, pulling from `e.responseJSON`.  Do we know the error is coming from a failed request or could it be some other exception thrown?  Maybe we should just capture the value of `e`?  Sentry would also be useful for this but if we don't use that we may want to try to extract e.responseJSON, e.msg, etc to capture additional debugging information.

Reply:
1. We do not have Sentry as of now, but it looks interesting to explore and consider for the future.
2. The error is coming from a failed request for the bug reported here but can be some other exception as well in some other cases. Thus for specifically tracking the bug reported here, I think it's fine to use state.error. Also, I am tracking it only via google analytics (and not our internal tracking) and the 'label' for that can be of type string and not object (e is an error object here).

Overall, we are tracking this bug because we are not able to reproduce it naturally without tweaking the browser cookies, so we want to track/detect if it occurs in prod naturally and how frequently.